eo-maven-plugin/src/main/java/org/eolang/maven/ResolveMojo.java

```diff
@@ -202,8 +214,40 @@ private Collection<Dependency> deps() throws IOException {
             .collect(Collectors.toList());
     }
 
+    /**
+     * Check if all dependencies have transitive dependencies.
+     *
+     * @param deps Dependencies
+     * @throws java.lang.IllegalStateException if a transitive dependency is found
+     */
+    private void checkTransitive(final Collection<Dependency> deps) {
+        if (this.checkTransitive) {
+            deps.forEach(
```

Comment:
@volodya-lombrozo compare this:

```
list.forEach(
  a -> {
    doit();
  }
)
```

with this:

```
for (list : a) {
  doit();
)
```

which is more compact and has less indentation?

Reply:
@yegor256 fixed.